src/dsp/rds/parser_impl.cc

```diff
@@ -103,16 +103,16 @@ void parser_impl::decode_type0(unsigned int *group, bool B) {
 	/* see page 41, table 9 of the standard */
 	switch (segment_address) {
 		case 0:
-			mono_stereo=decoder_control_bit;
+			static_pty=!decoder_control_bit;
```

Comment:
I think it may make sense to leave this bit as is, and change the display (in dockrds.cpp) instead. That way there won't be an API change in the PMT message format, which could potentially break applications that depend on gr-rds.

```suggestion
			static_pty=decoder_control_bit;
```

Reply:
Hmmm.
I've tried to preserve compatibility by keeping PMT message characters meaning unchanged.
This bit means "dynamic PTY" when set, but the corresponding PMT message character means "static PTY". That's why it is inverted here.

Reply:
> the corresponding PMT message character means "static PTY"

I suppose that may have been the intention, since the variable is named `static_pty`. But it could also be that the choice of variable name was a mistake, and it should have been named `dynamic_pty`.

I think the less confusing choice would be to copy the DI bits into the PMT message verbatim (as they are currently) and rename the variable.

In any case, it's gr-rds's API so we should probably see what @bastibl would prefer to do upstream.